tests/test_estimateRadialVelocity.py

```diff
@@ -0,0 +1,174 @@
+        flags = MaskHelper()
+
+        return PfsSimpleSpectrum(target, wavelength, flux, mask, flags)
+
+    def createObsSpectrum(self, refSpectrum, trueVelocity, snr):
+        """
+        Create a fake observed spectrum.
+
+        Parameters
+        ----------
+        refSpectrum : `pfs.datamodel.pfsSimpleSpectrum.PfsSimpleSpectrum`
+            Reference spectrum.
+        trueVelocity : `float`
+            Radial velocity of the observed object.
+        snr : `float
+            Signal-to-noise ratio.
+
+        Returns
+        -------
+        spectrum : `pfs.datamodel.pfsFiberArray.PfsFiberArray`
+        """
+        refWavelength = refSpectrum.wavelength
+        refFlux = refSpectrum.flux
+
+        # Select any 3nm range made of an odd number of sampling points
+        index = np.nonzero((800 <= refWavelength) & (refWavelength <= 803))[0]
+        index = index[: (len(index) + 1) // 2 * 2 - 1]
+        lsfWavelength = refWavelength[index]
+
+        lsfFlux = createLsf(lsfWavelength, lsfWavelength[len(lsfWavelength)//2])
+        lsfFlux /= np.sum(lsfFlux)
+
+        c = 299792.458
+        doppler = np.sqrt((1 + trueVelocity / c) / (1 - trueVelocity / c))
+
+        wavelength = np.linspace(400, 1200, 12000)
+        flux = scipy.signal.fftconvolve(refFlux, lsfFlux, mode="same")
+        flux = scipy.interpolate.interp1d(refWavelength*doppler, flux, kind="cubic")(wavelength)
+
+        variance = flux / (snr*snr)
+        flux = self.np_random.poisson(flux * (snr*snr)) / (snr*snr)
+
+        target = Target(0, 0, "0,0", 0)
+        observations = Observations(
+            np.zeros(0, dtype=int),
+            np.zeros(0, dtype="U0"),
+            np.zeros(0, dtype=int),
+            np.zeros(0, dtype=int),
+            np.zeros(0, dtype=int),
+            np.zeros(shape=(0, 2), dtype=float),
+            np.zeros(shape=(0, 2), dtype=float)
+        )
+        mask = np.zeros(len(wavelength), dtype=int)
+        sky = np.zeros(len(wavelength), dtype=float)
+        covar = np.zeros(shape=(3, len(wavelength)), dtype=float)
+        covar[0, :] = variance
+        covar2 = np.zeros(shape=(0, 0), dtype=float)
+        flags = MaskHelper()
+
+        return PfsFiberArray(target, observations, wavelength, flux, mask, sky, covar, covar2, flags)
+
+
+def createLsf(wavelength, peak):
+    """
+    Fake line spread function for observed spectra.
+    It is normalized so that its height is 1.
+
+    Parameters
+    ----------
+    wavelength : `numpy.array`
+        Wavelength in nm.
+    peak : `float` or `numpy.array`
+        Peak position in nm.
+        If this is `numpy.array`, it must be broadcastable to ``wavelength``
+        (For example, ``wavelength.shape == (1, N)``
+        and ``peak.shape == (M, 1)``,
+        in which case the return value has shape ``(M, N)``.)
+
+    Returns
+    -------
+    flux : `numpy.array`
+        An LSF whose peak is at `peak`.
+    """
+    a = 6.7
```

Comment:
Make this configurable.

You could use `pfs.drp.stella.GaussianLsf`?

Reply:
This function is named LSF but actually its return value is used to dig down a fake continuous spectrum to imitate an absorption line. Because of how it is implemented, it is necessary that the height of the returned peak is 1, unlike `GaussianLsf`.